committeeoversightapp/static/css/custom.css

```diff
@@ -10,6 +10,44 @@ body {
   padding: 30px;
 }
 
+.dynamic-form {
+  background: #e8eef9;
+  padding: 0px 20px 50px 20px;
+  margin-bottom: 10px;
+  border-radius: 5px;
+}
+
+.dynamic-form div div {
+  padding-top: 10px;
+}
+
+.dynamic-form div div input {
+  display: block;
+  width: 100%;
+  height: calc(2.25rem + 2px);
+  padding: .375rem .75rem;
+  font-size: 1rem;
+  line-height: 1.5;
+  color: #495057;
+  background-color: #fff;
+  background-clip: padding-box;
+  border: 1px solid #ced4da;
+  border-radius: .25rem;
+  transition: border-color .15s ease-in-out,box-shadow .15s ease-in-out;
+}
+
+.delete-row {
+  color: #b80026;
+  float: right;
+  padding: 15px 0px;
+  font-size: 14px;
+  font-weight: 500;
+}
+
+.delete-row:hover {
+  color: #6f0017;
+}
+
```

Comment:
custom styles - nice! I always find that I can minimize CSS-refactor-headaches by putting all colors inside variables: https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables and [an example](https://github.com/datamade/nyc-council-councilmatic/blob/master/nyc/static/css/city_custom.css#L1)

Reply:
love it, added!